### DISCUSSION TITLE
User Status Messages

### DISCUSSION THREAD

**Discussion by xmradio** [BearWare/TeamTalk5#2379] [Ideas]
Good evening everyone,
I was looking back and noticed that there was issues in the past regarding the user right for user changing status. However, with the SpamBot and other bots that kick usernames, this is an important feature to add for bots.
For example, it's good to add it to a MusicBot so that it doesn't get kicked by a spambot. Can you please tell us how we can enable this because I know it is possible to do in the server, because it was only removed from clients. Is there a special way to add it to the XML so that it will have it disabled?
We had thought about downlading the older version of TeamTalk that supported this feature, how ever it is not accessible with telling us if a user right is enabled or disabled with Jaws. I also know that it forced the mail gender, for me though this is not a problem with a bot.
@bear101 Could you please tell me how to enable this for my Musicbot account in my server?
Thank you.

**Comment by bear101** [comment]
BearWare.dk is not affiliated with Musicbot.
